common/dynamicconfig/constants.go

```diff
@@ -79,6 +79,9 @@ const (
 	EnableReadVisibilityFromES = "system.enableReadVisibilityFromES"
 	// EnableReadFromSecondaryAdvancedVisibility is the config to enable read from secondary Elasticsearch
 	EnableReadFromSecondaryAdvancedVisibility = "system.enableReadFromSecondaryAdvancedVisibility"
+	// VisibilityDisableOrderByClause is the config to disable ORDERY BY clause for Elasticsearch
+	VisibilityDisableOrderByClause = "system.VisibilityDisableOrderByClause"
```

Comment:
I know there is a mess, but majority uses camelCase:
```suggestion
	VisibilityDisableOrderByClause = "system.visibilityDisableOrderByClause"
```